pytest_localserver/__init__.py

```diff
@@ -1 +1 @@
-VERSION = '0.5.1'
+from pytest_localserver._version import version as VERSION
```

Comment:
Hmmm... this seems to be failing tests on some Python environments, including 3.5 and 3.6, because it claims there's no such module `pytest_localserver._version`. I can't reproduce it locally though.

Reply:
It's not failing anymore, so I guess either something fixed this (whatever was going on :shrug:) or it's intermittent, in which case all we can do is hope it comes back later.